<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmakeProjectManager::Internal::CustomWidgetPluginWizardPage</class>
 <widget class="QWizardPage" name="QmakeProjectManager::Internal::CustomWidgetPluginWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Plugin and Collection Class Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="subTitleLabel">
     <property name="text">
      <string>Specify the properties of the plugin library and the collection class.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="subTitleSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="collectionClassLabel">
         <property name="text">
          <string>Collection class:</string>
         </property>
         <property name="buddy">
          <cstring>collectionClassEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="collectionClassEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="collectionHeaderLabel">
         <property name="text">
          <string>Collection header file:</string>
         </property>
         <property name="buddy">
          <cstring>collectionHeaderEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="collectionHeaderEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="collectionSourceLabel">
         <property name="text">
          <string>Collection source file:</string>
         </property>
         <property name="buddy">
          <cstring>collectionSourceEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="collectionSourceEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="pluginNameLabel">
         <property name="text">
          <string>Plugin name:</string>
         </property>
         <property name="buddy">
          <cstring>pluginNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="pluginNameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="resourceFileLabel">
         <property name="text">
          <string>Resource file:</string>
         </property>
         <property name="buddy">
          <cstring>resourceFileEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="resourceFileEdit">
         <property name="text">
          <string>icons.qrc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
